python_modules/jupedsim/jupedsim/models/social_force.py

```diff
@@ -0,0 +1,148 @@
+from dataclasses import dataclass
+
+import jupedsim.native as py_jps
+
+
+@dataclass(kw_only=True)
+class SocialForceModel:
+    """Parameters for Social Force Model
+
+    All attributes are initialized with reasonably good defaults.
+
+
+    Attributes:
+        bodyForce: describes the strength with which an agent is influenced by pushing forces from obstacles and neighbors in its direct proximity.
+        friction: describes the strength with which an agent is influenced by frictional forces from obstacles and neighbors in its direct proximity.
+    """
+
+    bodyForce: float = 120000
+    friction: float = 240000
+
+
+@dataclass(kw_only=True)
+class SocialForceModelAgentParameters:
+    """
+    Parameters required to create an Agent in the Social Force Model.
+
+    Attributes:
+        position: Position of the agent.
+        orientation: Orientation of the agent.
+        journey_id: Id of the journey the agent follows.
+        stage_id: Id of the stage the agent targets.
+        velocity: current velocity of the agent
+        mass: mass of the agent.
+        desiredSpeed: desired Speed of the agent.
+        reactionTime: reaction Time of the agent.
+        agentScale: indicates how strong an agent is influenced by pushing forces from neighbors.
+        obstacleScale: indicates how strong an agent is influenced by pushing forces from obstacles.
+        forceDistance: indicates how much the distance between an agent and obstacles or neighbors influences social forces.
+        radius: radius of the space an agent occupies.
+    """
+
+    position: tuple[float, float] = (0.0, 0.0)
+    orientation: tuple[float, float] = (0.0, 0.0)
+    journey_id: int = -1
+    stage_id: int = -1
+    velocity: tuple[float, float] = (0.0, 0.0)
+    mass: float = 80.0
+    desiredSpeed: float = 0.8
+    reactionTime: float = 0.5
+    agentScale: float = 2000
+    obstacleScale: float = 2000
```

Comment:
Same as above, is there a scale/unit to these values?

Reply:
there is now a comment containing the unit, a reference to the paper where the values are from and in some cases additional notes